src/app/components/Byline/index.tsx

```diff
@@ -21,154 +22,172 @@ const Byline = ({
-                css={[BylineCss.author, BylineCss.authorLink]}
-                size="bodyCopy"
-                fontVariant="sansBold"
-              >
-                {authorName}
-              </Text>
-            </span>
-          )}
-        </li>
-        <li>
-          <span role="text">
-            <VisuallyHiddenText>{`${jobRoleTranslated}, `} </VisuallyHiddenText>
-            <Text css={BylineCss.jobRole} fontVariant="sansBold" size="brevier">
-              {jobRole}
-            </Text>
-          </span>
-        </li>
-        {twitterLink ? (
-          <li>
-            <a
-              css={[BylineCss.link, BylineCss.twitterLink]}
-              className="focusIndicatorReducedWidth"
-              href={twitterLink}
-              aria-labelledby="byline-twitter-link"
-            >
-              <span role="text" id="byline-twitter-link">
-                <VisuallyHiddenText lang="en-GB">
-                  {`Twitter, `}
-                </VisuallyHiddenText>
-                <Text
-                  className="byline__link-text"
-                  css={BylineCss.twitterText}
-                  size="brevier"
-                  fontVariant="sansBold"
-                >{`@${twitterText}`}</Text>
-                {isRtl ? (
-                  <LeftChevron css={BylineCss.twitterChevron} />
-                ) : (
-                  <RightChevron css={BylineCss.twitterChevron} />
-                )}
               </span>
-            </a>
+            )}
           </li>
-        ) : null}
-        {location ? (
           <li>
-            <span role="text" css={BylineCss.location}>
-              <Text
-                css={BylineCss.reportingFromText}
-                size="brevier"
-                fontVariant="sansRegularItalic"
-              >
-                {`${reportingFromTranslated} `}{' '}
-              </Text>
+            <span role="text">
+              <VisuallyHiddenText>
+                {`${jobRoleTranslated}, `}{' '}
+              </VisuallyHiddenText>
               <Text
-                css={BylineCss.locationText}
+                css={BylineCss.jobRole}
+                fontVariant="sansBold"
                 size="brevier"
-                fontVariant="sansRegular"
               >
-                {location}
+                {jobRole}
               </Text>
             </span>
           </li>
-        ) : null}
-        {children ? (
-          <li css={BylineCss.timestampLineBreak}>{children}</li>
-        ) : null}
-      </ul>
-    </section>
+          {twitterLink ? (
+            <li>
+              <a
+                css={[BylineCss.link, BylineCss.twitterLink]}
+                className="focusIndicatorReducedWidth"
+                href={twitterLink}
+                aria-labelledby="byline-twitter-link"
+              >
+                <span role="text" id="byline-twitter-link">
+                  <VisuallyHiddenText lang="en-GB">
+                    {`Twitter, `}
```

Comment:
Should this be X instead of Twitter?

Reply:
Maybe .... we had a ticket done for updating references to twitter, maybe this was missed intentionally? Conscious of the can of worms that this could be , unless we take the stance that we just update them as we see them? Dont want to confuse screen reader users by having it read out X in one place and twitter in others

Reply:
Let's check with Harry if he wants us to create a new ticket, and we can do so if he think it's worth it. Otherwise let's not change it as part of this work.

Reply:
the [Twitter to X](https://jira.dev.bbc.co.uk/browse/WORLDSERVICE-327) work only changed audience facing references to twitter. it was decided that other references to twitter within Simorgh(and our BFF module) which our users are never going to see would be left alone.

but at the time `VisuallyHiddenText` that contains 'X' wasn't considered so this is

Reply:
Harry said it would be worth doing, but after a quick search i _think_ this is the only place where twitter is inside visually hidden text, so i'll just update it as part of this.